src/hello.cpp

```diff
@@ -0,0 +1,10 @@
+#include "hello.hpp"
+
+// Standalone function (because it's not a class)
+NAN_METHOD(hello) {
+
+    // info comes from the NAN_METHOD macro, which returns differently
+    // according to the version of node
+    info.GetReturnValue().Set(Nan::New<v8::String>("world").ToLocalChecked());
+
+}
```

Comment:
Oh, and also wrapping the `NAN_METHOD(hello);` in the `hello.hpp` in a namespace.